unified-codes/apps/web/src/components/containers/explorer/ExplorerSearchBar.tsx

```diff
@@ -2,17 +2,45 @@ import * as React from 'react';
 import { batch, connect } from 'react-redux';
 
 import { SearchBar } from '@unified-codes/ui/components';
-import { withStyles } from '@unified-codes/ui/styles';
+import { makeStyles, createStyles } from '@unified-codes/ui/styles';
 
 import { ExplorerActions, IExplorerAction } from '../../../actions';
 import { ExplorerSelectors } from '../../../selectors';
 import { IState } from '../../../types';
 import { ITheme } from '../../../styles';
 
-const styles = (_: ITheme) => ({
-  input: { paddingLeft: 15 },
-  button: { marginTop: 15 },
-});
+const useStyles = makeStyles((theme: ITheme) =>
+  createStyles({
+    root: { color: theme.palette.text.primary },
+    input: { paddingLeft: 15 },
```

Comment:
```suggestion
    input: { paddingLeft: 15, color: '#a2a2a2' },
```

I find the white text strange

![Screen Shot 2021-05-25 at 9 51 50 AM](https://user-images.githubusercontent.com/9192912/119413745-67afd580-bd42-11eb-9a63-a34c88a3aebe.png)

and would suggest adding the colour here, and then updating `ClearInput` to use the style. That results in:

![Screen Shot 2021-05-25 at 10 18 13 AM](https://user-images.githubusercontent.com/9192912/119413857-93cb5680-bd42-11eb-8158-4bed7f06f150.png)

and

![Screen Shot 2021-05-25 at 10 18 20 AM](https://user-images.githubusercontent.com/9192912/119413881-9cbc2800-bd42-11eb-966e-3d77a70c8b06.png)

Reply:
Thanks, the fading of the label was a bit odd, this is much nicer :) ee895df5a